kolibri/core/logger/api.py

```diff
@@ -823,8 +823,7 @@ def _update_summary_log(
         summarylog = ContentSummaryLog.objects.get(
             content_id=sessionlog.content_id, user=user
         )
-        was_complete = summarylog.progress >= 1
```

Comment:
I think I would prefer we not make this change - as I think it makes sense for the summary log's completion timestamp to be updated to the latest completion.

In the only circumstance where this can happen, a practice quiz with multiple attempts, we still have the complete history of completion_timestamps on the MasteryLogs associated, but the final ContentSummaryLog completion_timestamp is the most recent (this is different to the notification though, as there we are noting the _first_ time it was completed).

Reply:
All right, I will check the MasteryLogs and see how we can query it to get the first timestamp